es/build.gradle

```diff
@@ -7,21 +7,23 @@ repositories {
 
 tasks.withType(JavaCompile) {
     options.compilerArgs << '-XDignore.symbol.file'
-    options.fork = true // may not needed on 1.8
-    options.forkOptions.executable = 'javac' // may not needed on 1.8
+    options.fork = true // probably not required when using java 1.8
+    options.forkOptions.executable = 'javac'
 }
 
 // generated in upstream directory with:
 // mvn dependency:tree | grep -o ' .- .*:compile' | cut -d ' ' -f 3 | sort | sed "s/\(.*\):jar:\(.*\):compile/compile '\1:\2'/g"
 dependencies {
 
+    // if you update the versions here
+    // also update the dependencies in client/build.gradle
```

Comment:
Can't we add a version property to settings.gradle or somewhere and re-use that across `client/build.gradle` and here?